source/code-snippets/authentication/aws.js

```diff
@@ -7,24 +7,26 @@ const clusterUrl = "<MongoDB cluster url>";
 
 const authMechanism = "MONGODB-AWS";
 
-// Replace the following with your MongoDB deployment's connection string.
-const uri =
+let uri =
   `mongodb+srv://${accessKeyId}:${secretAccessKey}@${clusterUrl}/?authMechanism=${authMechanism}`;
 
-// Create a new MongoClient
+// Uncomment the following lines if your AWS authentication setup requires a session token.
+// const sessionToken = encodeURIComponent("<AWS_SESSION_TOKEN>");
+// uri = uri.concat(`&authMechanismProperties=${sessionToken}`);
```

Comment:
I don't have a strong preference because it's correct and (I think) idiomatic JS, but another common way to accomplish this is:
```
uri += `&authMechanismProperties=${sessionToken}`;
```